### ISSUE TITLE
Make colliders and other relevant components scale when scaling the entity's `Transform`

### ISSUE THREAD

**Issue by sunreef** [amethyst/evoli#53]
Right now, if you change the scale of an entity via the `set_scale`method of its `Transform`, the collider `Circle` does not change radius. 

If we can adapt the size of the collider and other relevant components, this would let us create creatures of the same type but different sizes: parents and children.

**Comment by marotili** [comment]
I don't think it is worth the effort tackling this atm, because the collision system will be replaced by a real one as soon as physics integration is done. I would imagine that this will be supported then
